server/incoming/request.go

```diff
@@ -20,169 +20,139 @@ type SessionService interface {
 }
 
 type Request struct {
-	ctx context.Context
-
+	ctx            context.Context
 	mm             *pluginapi.Client
 	config         config.Service
 	Log            utils.Logger
 	sessionService SessionService
+	requestID      string
 
-	requestID string
-	pluginID  string
+	dest apps.AppID
 
-	appID                 apps.AppID
```

Comment:
replaced with specific `sourceAppID` and `dest` (rather than `destAppID` - for brevity, it is used a lot)